client/src/modules/stock/lots/registry.html

```diff
@@ -35,12 +35,12 @@
             </li>
             <li role="separator" class="divider"></li>
             <li role="menuitem">
-              <a ng-href="/reports/stock/lots/?{{ StockLotsCtrl.download('lot', 'pdf') }}" download="{{ 'VOUCHER_REGISTRY.TITLE' | translate }}">
+              <a ng-href="/reports/stock/lots/?{{ StockLotsCtrl.getQueryString('lot', 'pdf') }}" download="{{ 'VOUCHER_REGISTRY.TITLE' | translate }}">
```

Comment:
Why is this download called 'VOUCHER_REGISTRY.TITLE'?  I know if pre-existed your changes, but can you fix that?